### ISSUE TITLE
synctus.com is down

### ISSUE THREAD

**Issue by retorquere** [basak/ddar#7]

**Comment by estan** [comment]
Any news on this. What has happened?

**Comment by basak** [comment]
Synctus itself is defunct now, although ddar (which grew out of Synctus) lives on. The host running the website is in dire need of security updates, so after suspicious activity I disabled everything and haven't got round to putting the ddar page back up somewhere else.

Can you tell me what is missing that you cannot get from this Github repo please? I'm thinking about just not bothering with a website and using the Github page as the primary location instead.

**Comment by estan** [comment]
Ah. I see. I was just curious and tried to follow the

```
For details, see: http://www.synctus.com/ddar/.
```

link in README.markdown.

The README contains no information on how ddar works or how to use it, but I guess I can build the manpage and look at that :)

**Comment by basak** [comment]
> The README contains no information on how ddar works or how to use it...

Oh, thanks. I will need to fix that.

For now, how about: http://web.archive.org/web/20131209161307/http://www.synctus.com/ddar/

**Comment by estan** [comment]
Right, had forgot about archive.org. Thanks!

**Comment by estan** [comment]
I've now read it, and the information there is great. Would be good to bring it over as wiki pages here on GitHub, or as a GitHub Pages site, including the comparisons with tarsnap/rsync-backup.

If you could enable the wiki or generate a GitHub Pages site through the repo Settings, I could help with bringing the content over from archive.org if you want.

**Comment by alexreg** [comment]
Agreed with @estan, above. I too would help migrating over to a GitHub Pages site or wiki, if you fancy.

**Comment by alexreg** [comment]
FYI, since the web pages are clean and simple, it would probably be very straightforward indeed to convert them to Markdown format and migrate them here.

**Comment by alexreg** [comment]
Okay, I was bored so I went ahead and did the conversion to wiki for you! Here it is in my fork: https://github.com/alexreg/ddar/wiki

You can just pull from the repo for that wiki, to get it on the official project, I believe.

**Comment by basak** [comment]
@alexreg 

Pulled into here. Thanks!

I'll leave this issue open to remind me to set up `www.synctus.com/ddar` somewhere else and redirect to the wiki.

**Comment by alexreg** [comment]
No problem. And yes, sounds like a good idea to do that redirect. I might have left in one or two links to the Synctus website in the wiki too; I didn’t know how you wanted to alter those sentences.
